components/automate-ui/src/app/entities/api-tokens/api-token.effects.ts

```diff
@@ -39,125 +39,140 @@ export class ApiTokenEffects {
     private requests: ApiTokenRequests
   ) { }
 
-  @Effect()
-  getAllTokens$ = this.actions$.pipe(ofType<GetAllTokens>(ApiTokenActionTypes.GET_ALL),
+  getAllTokens$ = createEffect(() => {
+    return this.actions$.pipe(ofType<GetAllTokens>(ApiTokenActionTypes.GET_ALL),
     mergeMap((_action) =>
       this.requests.getAll().pipe(
         map(resp => new GetAllTokensSuccess(resp)),
         catchError((error: HttpErrorResponse) => of(new GetAllTokensFailure(error))))
     ));
+  });
```

Comment:
You'll see this new format used throughout all our effects files moving forward and its how we should continue to compose effects.